home.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:io';
+import 'App.dart';

```

Comment:
again, you're doing a circular import which is not recommended.
Home imports App, and App imports Home.
Please, make the start method of Home returns the UserOption as a regular return or a closure/callback instead.